web/src/components/table/use-cases/models.tsx

```diff
@@ -222,13 +228,12 @@ export default function ModelTable({ projectId }: { projectId: string }) {
       accessorKey: "actions",
       header: "Actions",
       cell: ({ row }) => {
-        return row.original.maintainer === "User" ? (
+        return (
```

Comment:
I agree that showing disabled buttons for users without sufficient roles is an improvement here.

Do you think the UX is better when showing a disabled delete button on langfuse-provided models even though they cannot be deleted?

Reply:
It should be signified in the UI in this case. Another way would be to present an info icon and once clicked (or hovered) it would explain that built-in models are non-removable.

Reply:
maybe a simple `title="Built-in models cannot be deleted"`?

Reply:
yep, good enough for me

Reply:
in general the screen deserves some more love - I'll follow up with another PR

Reply:
hey @marcklingen are you waiting for me here? let me know if we're ready to move with this pr ;)

Reply:
sorry, just got to this as today was super busy, thank you!

Reply:
> in general the screen deserves some more love - I'll follow up with another PR

agree, this is one of the most complex tables right now. Some change to the price calculation logic are necessary soon, but the underlying logic will only become more complex.

One option would be to render the groups of models more nicely as the full list including changes over time are hardly interesting and could be moved to separate pages.